src/components/TextField/TextField.tsx

```diff
@@ -310,14 +310,18 @@ class TextField extends React.PureComponent<CombinedProps, State> {
       describedBy.push(`${id}CharacterCounter`);
     }
 
-    const labelledBy = [labelID(id)];
+    const labelledBy: string[] = [];
     if (prefix) {
       labelledBy.push(`${id}Prefix`);
     }
     if (suffix) {
       labelledBy.push(`${id}Suffix`);
     }
 
+    if (labelledBy.length) {
+      labelledBy.unshift(labelID(id));
```

Comment:
I realize this is literally putting the text field label in the same position as it currently would be, but looking at this has me wondering a couple of things @dpersing:

- Does the order matter when a field is labelled by multiple things? In other words, do screen readers read the label text, prefix text content (if any) and the suffix text (if any)? Or do they read the `aria-labelledby` value itself word for word to describe the field?

Reply:
>Does the order matter when a field is labelled by multiple things?

The `id` values are supposed to set the reading order, but since we're kind of doubling up on those reading the `<label>` first seems to be the default. Generally AT will default to reading HTML native features if they're available, then read ARIA features. Generally. In theory. 😂